lib/v2/sduwh/extractor/news.js

```diff
@@ -0,0 +1,15 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (link, ctx) =>
+    await ctx.cache.tryGet(link, async () => {
+        try {
+            const result = await got.get(link);
```

Comment:
```suggestion
            const result = await got(link);
```

Reply:
Fixed now. Thanks for pointing out this.